src/context/extensions/steps/getExtensions.js

```diff
@@ -191,6 +192,19 @@ function getCompleteExtensionTree(type, roc, path, initialState) {
     );
 }
 
+function getNearestParents(roc) {
+    return [
+        ...(roc.packages || []),
+        ...(roc.plugins || []),
+    ].map((parent) => {
```

Comment:
Correct, it comes from this in a typical Roc extension.

```
packages: [
  require.resolve('roc-package-webpack-node-dev'),
  require.resolve('roc-package-webpack-web-dev'),
],
plugins: [
  require.resolve('roc-plugin-browsersync'),
  require.resolve('roc-plugin-style-css'),
  require.resolve('roc-plugin-assets-images'),
]
```

What do you mean with npm notation?